pftp/proxy.go

```diff
@@ -184,18 +183,20 @@ func (s *proxyServer) GetConn() net.Conn {
 func (s *proxyServer) SetDataHandler(handler *dataHandler) {
 	// only one data connection available in same time.
 	if s.dataConnector != nil {
-		// if already had previous data handler, wait until response sent to client.
-		for s.waitDataResponse {
-			time.Sleep(time.Millisecond * 500)
+		// if already had previous data handler in use, wait until end.
+		if s.dataConnector.needWait {
```

Comment:
also, could you please consider good name because `waitTransferEnd` and `needWait` mean is very close.

Reply:
😅 

Roger that!

Reply:
Renamed.
New name will describe state of datahandler attached to proxyhandler because attached means need to wait previous datahandler job end.